backend/main.py

```diff
@@ -0,0 +1,68 @@
+from CSPEC_Parser import CSPEC_Parser
```

Comment:
is main the right name for this file? and also no author at the top, but this is just very small thing